### PR TITLE
Tanner develop fixes phase #2

### PR DESCRIPTION
**Pull request by viskey98**
This will fix the following:
- JSON error due to different python version.
- NoneType user-agent error.

### CODE REVIEW

File: tanner/api/api.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def __init__(self, redis_client):
             if not query_res:
                 return 'Invalid SNARE UUID'
             for (i, val) in enumerate(query_res):
-                query_res[i] = json.loads(val)
+                query_res[i] = json.loads(val.decode("utf-8"))
```
Comment (afeena) [inline tanner/api/api.py]: I'd better put encoding things into redis query

Comment (viskey98) [reply tanner/api/api.py]: `zrevrangebyscore` redis query doesn't have encoding parameter. So except for this case, I have added encoding things in the redis query itself. Please look into it.

Comment (afeena) [reply tanner/api/api.py]: Actually it has this parameter (maybe they forgot to add it into the docs) :)

**Comment by viskey98** [comment]
@afeena That redis encoding is fixed. Please look into it now :)

**Pull request merged**